R/plate_plot.R

```diff
@@ -515,5 +522,9 @@ plate_plot <- function(data,
       panel.border = ggplot2::element_rect(size = stroke_width)
     )
 
-  plot
+  # if (display_plot) {
+  #   plot
+  # }
+
+  return(plot)
```

Comment:
This should not make a differences since it is the end of the function.

Reply:
I just tried it again with and without `return(plot)` and it does make a difference. The following pseudocode results in an empty plot being returned / saved:

```r
my_plot <- plate_plot(...)
ggsave(file, my_plot, ...)
```

I am however not sure why I first added the `if()` condition and then commented it out. It must have been by accident and I have now reverted the commenting out part of the commit.